### DISCUSSION TITLE
There's no support available for the dart language (Flutter)

### DISCUSSION THREAD

**Discussion by GiuseppeCrBrandi** [deepgram/community#65] [General help] [answered]
I'm trying to integrate the Deepgram API to my Flutter application but I found no documentation on how to do that.

**Comment by michaeljolley** [comment]
Hi @GiuseppeCrBrandi, we don't have documentation specifically for Flutter or Dart, but we do have a [blog post](https://blog.deepgram.com/flutter-speech-to-text-tutorial/) that describes how to do it.

**Reply by GiuseppeCrBrandi** [reply]
It doesn't show you how to do it from a pre recorded Audio File, for example a wav file. Only by Speech Generated Audio recorded by your microphone.

**Reply by BekahHW** [reply]
@GiuseppeCrBrandi have you checked out our documentation for pre-recorded audio? Let us know if you've tried to implement it using the blogpost and docs and if you're having issues.

**Comment by BekahHW** [comment]
@GiuseppeCrBrandi does that help or do you have more questions?

**Reply by GiuseppeCrBrandi** [reply]
It doesn't show you how to do it from a pre recorded Audio File, for example a wav file. Only by Speech Generated Audio recorded by your microphone.

**Comment by faizank09855** [comment, marked as answer]
hii ,
@GiuseppeCrBrandi you can follow this =>
https://dev.to/sbis04/decifer-generate-transcripts-with-ease-5hl3

**Comment by tempo-riz** [comment]
Hey I created a small dart/flutter package avalaible for every platform :)
https://pub.dev/packages/deepgram_speech_to_text
